src/Pages/hoonic/Main/components/maincontainer/Article/Article.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Comment from '../../Comments/Comment';
+// import COMMENT from '../../Comments/CommentData';
+import './Article.scss';
+
+class ArticleHoonic extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentValue: '',
+      commentList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/hoonic/commentData.json', {
+      method: 'Get',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          commentLsit: data,
+        });
+      });
+  }
+  //   this.setState({
+  //     commentList: COMMENT,
+  //   });
+
+  addCommentEnter = e => {
+    if (this.state.commentValue.length >= 1 && e.key === 'Enter') {
+      this.addComment();
+    } else {
+      this.setState({ commentValue: e.target.value });
+    }
+  };
+
+  addComment = () => {
+    const { commentList, commentValue } = this.state;
+    this.setState({
+      commentList: commentList.concat([commentValue]),
+      commentValue: '',
+    });
+  };
+
+  // this.state.commentList.concat([this.state.commentValue])
+  // commentList: [...commentList, commentValue] 스프레이드 연산자
+
+  render() {
+    const { commentValue, commentList } = this.state;
+    return (
+      <article>
+        <div className="feedHeader">
+          <img alt="profile img" src="/images/hoonic/profileicon.png" />
+          <span className="feedHeaderText">kancho</span>
+        </div>
+        <div className="feedImage">
+          <img alt="feefimg" src="/images/hoonic/feed_image.jpeg" />
+        </div>
+        <div className="iconBar">
+          <img
+            alt="heart"
+            className="iconBarHeart"
+            src="/images/hoonic/heart.png"
+          />
+          <img
+            alt="Chat"
+            className="iconBarChat"
+            src="/images/hoonic/free-icon-speech-bubble-13522.png"
+          />
+          <img alt="dm" className="iconBarDm" src="/images/hoonic/dm.png" />
+          <img
+            alt="book mark"
+            className="iconBarBookMark"
+            src="/images/hoonic/free-icon-bookmarks-1174410.png"
+          />
+        </div>
+        <p className="likeCount">좋아요 55,555개</p>
+        <div divclassName="commentLists">
+          <div className="chatList">
+            <span className="commentID">hoonic</span>
+            <span>안녕하세요</span>
+            {commentList.map(el => {
+              return (
+                <Comment
+                  // comment={el}
+                  key={el.id}
+                  name={el.userName}
+                  content={el.content}
+                />
+              );
+            })}
```

Comment:
```suggestion
            {commentList.map(el => (
                <Comment
                  // comment={el}
                  key={el.id}
                  name={el.userName}
                  content={el.content}
                />
              );
            )}
```
이런식으로 arrow function 에서 바로 return 하실 경우 중괄호랑 return keyword 생략 가능합니다!